src/test/fuzz/util/wallet.h

```diff
@@ -59,8 +59,9 @@ struct FuzzedWallet {
                 WalletDescriptor w_desc{std::move(parsed_desc), /*creation_time=*/0, /*range_start=*/0, /*range_end=*/1, /*next_index=*/0};
                 assert(!wallet->GetDescriptorScriptPubKeyMan(w_desc));
                 LOCK(wallet->cs_wallet);
-                auto spk_manager{wallet->AddWalletDescriptor(w_desc, keys, /*label=*/"", internal)};
-                assert(spk_manager);
+                auto spkm{wallet->AddWalletDescriptor(w_desc, keys, /*label=*/"", internal)};
+                assert(spkm);
+                auto spk_manager = spkm.value();
```

Comment:
We are no longer asserting that `spk_manager!=nullptr`.
Could unify these lines + the missing check:
```c++
auto spk_manager = *Assert(wallet->AddWalletDescriptor(w_desc, keys, /*label=*/"", internal));
assert(spk_manager);
```